### PR TITLE
Update build

### LINKED ISSUE #5853: Work-around for Inspector: The embedded browser failed to load
Hi. When I try to view the Flutter Inspector I get the error "The embedded browser failed to load". The processor I use is an Apple M1.
Thank you for your reply.

## Version info

<details>
<summary>flutter doctor -v</summary>
[✓] Flutter (Channel stable, 2.5.3, on macOS 12.0.1 21A559 darwin-arm, locale en-RU)
    • Flutter version 2.5.3 at /Users/user/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 18116933e7 (3 weeks ago), 2021-10-15 10:46:35 -0700
    • Engine revision d3ea636dc5
    • Dart version 2.14.4

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/user/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 13.1, Build version 13A1030d
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.61.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.27.0

[✓] Connected device (2 available)
    • sdk gphone arm64 (mobile) • emulator-5554 • android-arm64  • Android 11 (API 30) (emulator)
    • Chrome (web)              • chrome        • web-javascript • Google Chrome 95.0.4638.69

• No issues found!
</details>
<details>
<summary>Android Studio Plugin versions</summary>

</details>

### PR DESCRIPTION
**Pull request by stevemessick**
@jwren 

Adds a build spec for IntelliJ beta, which does not work (so is dev channel only).

Updates survey URL.

Fixes #5853

### CODE REVIEW

File: flutter-idea/src/io/flutter/survey/FlutterSurveyService.java [hunk 1]
```diff
@@ -20,12 +20,11 @@
 public class FlutterSurveyService {
   private static final String FLUTTER_LAST_SURVEY_CONTENT_CHECK_KEY = "FLUTTER_LAST_SURVEY_CONTENT_CHECK_KEY";
   private static final long CHECK_INTERVAL_IN_MS = TimeUnit.HOURS.toMillis(40);
-  private static final String CONTENT_URL = "https://flutter.dev/f/flutter-survey-metadata.json";
+  private static final String CONTENT_URL = "https://docs.flutter.dev/f/dart-devtools-survey-metadata.json";
```
Comment (kenzieschmoll) [inline flutter-idea/src/io/flutter/survey/FlutterSurveyService.java:23]: The IDEs should still be using flutter-survey-metadata.json, so you'll just need to add the `docs.` here

Comment (stevemessick) [reply flutter-idea/src/io/flutter/survey/FlutterSurveyService.java]: Thanks, updated.

**Review by kenzieschmoll — APPROVED** [review]
LGTM on survey URL changes

**Comment by stevemessick** [comment]
Thanks @kenzieschmoll -- I'll merge this today, but will give @jwren a chance to check the beta build spec.

**Review by jwren — APPROVED** [review]

**Pull request merged**